src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,46 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
```

Comment:
In my opinion, health should be introduced as a function parameter.
I'm talking about constructor(name, health = 100) {...

Reply:
I would argue, it's always 100 so both ways are correct imo.